src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -35,18 +35,34 @@ method groovy.lang.Closure setDelegate java.lang.Object
 method groovy.lang.Closure setResolveStrategy int
 method groovy.lang.GString plus java.lang.String
 method groovy.lang.Script getBinding
+method groovy.util.Node get java.lang.String
+method groovy.util.NodeList getAt java.lang.String
+
+#There are other issues whith these classes than the whitelisting, but should at least be here
+new groovy.util.XmlParser
+new groovy.util.XmlSlurper
+method groovy.util.XmlParser parseText java.lang.String
+method groovy.util.XmlSlurper parseText java.lang.String
+new groovy.xml.MarkupBuilder java.io.Writer
 
 # Needed for Groovy Templates to emit output:
 method java.io.PrintWriter print java.lang.Object
 method java.io.PrintWriter print java.lang.String
 
 # Useful general stuff:
 new java.io.StringReader java.lang.String
+new java.io.StringWriter
+new java.lang.Boolean java.lang.String
+staticMethod java.lang.Boolean parseBoolean java.lang.String
+staticMethod java.lang.Boolean valueOf boolean
+staticMethod java.lang.Boolean valueOf java.lang.String
 method java.lang.CharSequence charAt int
 method java.lang.CharSequence length
+field java.lang.Class name
```

Comment:
I think this was for `whatever.class.name` hence the field instead of `getName()`